app/_hub/kong-inc/rate-limiting/index.md

```diff
@@ -161,7 +179,8 @@ params:
 
 ## Headers sent to the client
 
-When this plugin is enabled, Kong will send some additional headers back to the client telling what are the limits allowed, how many requests are available and how long it will take until the quota will be restored, for example:
+When this plugin is enabled, Kong will send some additional headers back to the client telling what are the limits allowed, how many requests are available,
+and how long it will take until the quota will be restored. For example:
```

Comment:
why the line break here?

Reply:
> why the line break here?

good eyes and good point. was trying to line break at 80 chars and missed that. Now that I see this text, it is in need of editing.

Reply:
This PR is scope creeping a bit. The original intent was to add datatypes, but the more I look at the docs, the more errors I find. Made some additional corrections in commit e837451. Edited for active rather than future/passive (for example, "will be") tense.